### PR TITLE
#167780198 fetch categories and create article fix

### PR DESCRIPTION
**Pull request by truestbyheart**
#### What does this PR do?
* It enables the user to fetch categories.
* It removes the image upload  functionality, this function is being performed on the front-end

#### Description of Task to be completed?
 * Create end-point for fetching categories
#### How should this be manually tested?
* Clone the repo to the local machine.
* Run ``npm install``   to install the dependencies.
* Set up the   ``.env`` as instructed by the   ``.env-example``
* Run  ``npm setup:dev``.
*Run ``npm start``.
* Send a GET request to ``/api/category``.
#### What are the relevant pivotal tracker stories?
#[167780198](https://www.pivotaltracker.com/story/show/167780198)

### CODE REVIEW

File: src/api/routes/index.js [hunk 1]
```diff
@@ -24,4 +26,6 @@ router.use('/notifications', notificationRouter);
 router.use('/termsAndConditions', termsAndConditionsRouter);
 router.use('/comment', reactionsRouter);
 
+router.use('/', categoryRouter);
```
Comment (alainburindi) [inline src/api/routes/index.js]: can you give this route a name? eg: `/categories`

File: src/api/routes/categoryRoute.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import categoryController from '../controllers/categoryController';
+
+const categoryRouter = new Router();
+
+categoryRouter.get('/category', categoryController.getCategory);
```
Comment (alainburindi) [inline src/api/routes/categoryRoute.js]: then this would be something like `/all`

File: tests/category.test.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+chai.should();
+
+describe('GET category', () => {
+  it('should return list of categories', (done) => {
+    chai
+      .request(server)
+      .get('/api/category')
+      .end((err, res) => {
+        res.body.should.have.status(status.OK);
+        res.body.categories.should.be.an('array');
```
Comment (alainburindi) [inline tests/category.test.js:17]: can you expect data?

File: src/api/controllers/categoryController.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import models from '../models';
+import status from '../../helpers/constants/status.codes';
+
+const { Category } = models;
+/**
+ * @class
+ */
+export default class categoryController {
```
Comment (alainburindi) [inline src/api/controllers/categoryController.js]: class name starts with a capital letter

**Review by alainburindi — CHANGES_REQUESTED** [review]

File: src/api/routes/categoryRoute.js (hunk 2 above)
Comment (premlogic) [reply src/api/routes/categoryRoute.js]: Why not just '/' here @alainburindi

File: src/api/routes/index.js (hunk 1 above)
Comment (premlogic) [reply src/api/routes/index.js]: I agree with @alainburindi @truestbyheart

**Review by nkpremices — COMMENTED** [review]
Good job @truestbyheart. However, I agree with what @alainburindi said for the route.

**Review by alainburindi — APPROVED** [review]

**Pull request merged**